Library/Formula/mtr.rb

```diff
@@ -12,11 +13,7 @@ class Mtr < Formula
     sha1 "be91d5c1ad604d190ef1e1d56842592b816197bf" => :mountain_lion
   end
 
-  head do
-    url "https://github.com/traviscross/mtr.git"
-    depends_on "automake" => :build
-  end
-
+  depends_on "automake" => :build
```

Comment:
The `system "./bootstrap.sh"` below is run only if `build.head?`, should it run unconditionally then?